scripts/qa_crowbarsetup.sh

```diff
@@ -529,11 +529,12 @@ function onadmin_prepare_sles12_repos()
 
     if ! $longdistance ; then
         add_mount "" "clouddata.cloud.suse.de:/srv/nfs/suse-12.0/install" "${targetdir_install}"
-    fi
 
-    for REPO in $sles12repolist ; do
-        add_mount "" "clouddata.cloud.suse.de:/srv/nfs/repos/$REPO" "$tftpboot_repos12_dir/$REPO"
-    done
+        for REPO in $sles12repolist ; do
+            add_mount "" "clouddata.cloud.suse.de:/srv/nfs/repos/$REPO" \
```

Comment:
Neither.  I am testing with `localreposdir_target` myself (remember NFS over WAN is not nice).  I have later commits which fix `localreposdir_target` for SLE12 but I can't submit PRs until these get merged.